README.md

```diff
@@ -7,12 +7,12 @@ The Endpoint Querier takes one arguement, a JSON file containing the endpoints w
 
 ```bash
 go get ./...
-go install ./...
+go mod download
```

Comment:
running `go mod download` gave me this output:
```
warning: pattern "all" matched no module dependencies
```

Are there additional setup steps that i need to do to get it to pay attention to go.mod?